src/components/AccordionList.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import { useTestIds } from "..";
+import { Accordion, AccordionProps, AccordionSize } from "./Accordion";
+
+interface AccordionListProps {
+  accordions: AccordionProps[];
+  /** Allows multiple accordions to be expanded simultaneously (enabled by default) */
+  allowMultipleExpanded?: boolean;
+  size?: AccordionSize;
+}
+
+export function AccordionList({ accordions, size, allowMultipleExpanded = true }: AccordionListProps) {
+  const [expandedIndex, setExpandedIndex] = useState<number>();
+  const tid = useTestIds({}, "accordionList");
+
+  return (
+    <>
+      {accordions.map((accordionProps, index, arr) => (
```

Comment:
**question:** I have never used the third parameter in the `map` function. I'm curious if there is a benefit to using the third param (`arr`) vs referencing `accordions` when you check `bottomBorder={index === arr.length - 1}`. I'm guessing you'd typically use this if the map function was in a separate scope and didn't have access to the variable you're `.map`-ing over. 
Do you happen to know if there is a benefit to using one over the other if the `map`'d variable is in scope?

Reply:
That's a feedback I got from Tyler a while back, I guess it makes it more succinct that you're re-using that particular array you're mapping. I'm not very opinionated on the matter, probably comes down to personal preferences (and some very specific edge cases)